sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -341,6 +341,31 @@ class CachedTableSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("tungsten cache uncompressed table and read") {
```

Comment:
Could reduce some duplication here by using a loop / ScalaTest's ability to have programatic test definitions.